scss/tech.scss

```diff
@@ -0,0 +1,54 @@
+.tech {
+  padding: 10px 40px;
+}
+
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  flex-basis: 150px;
+  left: 10px;
```

Comment:
I'd leave the content in it's original place and only move the border in the ::before